libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -184,6 +184,30 @@ void ReferencesResolver::endVisit(IdentifierPath const& _path)
 	_path.annotation().pathDeclarations = std::move(declarations);
 }
 
+bool ReferencesResolver::visit(UsingForDirective const& _usingFor)
+{
+	for (ASTPointer<IdentifierPath> const& path: _usingFor.functionsOrLibrary())
+	{
+		vector<Declaration const*> declarations = m_resolver.pathFromCurrentScopeWithAllDeclarations(path->path(), true /* _includeInvisibles */);
+		if (declarations.empty())
+		{
+			m_errorReporter.fatalDeclarationError(9589_error,
+				path->location(),
+				"Library or function name expected, but the identifier was not found or is not unique."
```

Comment:
Now that we handle this case separately, we could make the message more accurate. You can use `_usingFor.usesBraces()` to check if it's supposed to be a library or a function.